framework/math/petsc_utils/petsc_utils.cc

```diff
@@ -98,7 +98,8 @@ CreateCommonKrylovSolverSetup(Mat ref_matrix,
                               const std::string& in_solver_name,
                               const std::string& in_solver_type,
                               const std::string& in_preconditioner_type,
-                              double in_relative_residual_tolerance,
+                              double in_rel_tol,
+                              double in_abs_tol,
```

Comment:
Just making note of this for future reference: We should get rid of the `in_<var>` argument names.

Reply:
Rough search shows about 300 occurrences. Opened issue #173 for this task.